src/gui/attributetable/qgsattributetablemodel.cpp

```diff
@@ -673,19 +669,7 @@ QVariant QgsAttributeTableModel::headerData( int section, Qt::Orientation orient
 QVariant QgsAttributeTableModel::data( const QModelIndex &index, int role ) const
 {
   if ( !index.isValid() || !mLayer ||
-       ( role != Qt::TextAlignmentRole
-         && role != Qt::DisplayRole
-         && role != Qt::ToolTipRole
-         && role != Qt::EditRole
-         && role != FeatureIdRole
-         && role != FieldIndexRole
-         && role != Qt::BackgroundRole
-         && role != Qt::ForegroundRole
-         && role != Qt::DecorationRole
-         && role != Qt::FontRole
-         && role < SortRole
-       )
-     )
+       ( role != Qt::TextAlignmentRole && role != Qt::DisplayRole && role != Qt::ToolTipRole && role != Qt::EditRole && role != FeatureIdRole && role != FieldIndexRole && role != Qt::BackgroundRole && role != Qt::ForegroundRole && role != Qt::DecorationRole && role != Qt::FontRole && role < SortRole ) )
```

Comment:
revert? this makes it much less readable

Reply:
TBH I don't know which one of my editors made this unwanted mess ... I suspect vscode that I used to fix conflicts.